src/util.ts

```diff
@@ -20,3 +24,28 @@ export function getVersionFromToolcachePath(toolPath: string) {
 
   return toolPath;
 }
+
+export async function setupFromJdkFile(toolPath: string) {
```

Comment:
setupFromJdkFile -> extractJdkFile

Also, could you please add optional parameter `extension` argument here?
So we can reuse this function in other providers

Reply:
Done. Thank you.